src/sentry/templates/sentry/choose-project.html

```diff
@@ -0,0 +1,32 @@
+{% extends "sentry/bases/modal.html" %}
```

Comment:
Should this be a React page? I don't have enough context here, but seems like we don't want to add more HTML pages.

/cc @benvinegar @dcramer